### PR TITLE
Add Sample data to src/main/java/seedu/address/model/util/SampleDataUtil.java

### LINKED ISSUE #61: As a new user, I can have example client data already present in the app so that I can have a reference or play around in the app to get more familiar with it.

### PR DESCRIPTION
**Pull request by lilozz2**
As requested by @zhyuhan

### CODE REVIEW

**Comment by zhyuhan** [comment]
Thanks for this! Shall we add some interaction data to some of clients too?

**Review by zhyuhan — APPROVED** [review]
LGTM! Thanks!

File: src/main/java/seedu/address/model/person/Person.java [hunk 1]
```diff
@@ -259,6 +259,14 @@ public PersonBuilder withInteractions(Set<Interaction> interactions) {
...
         }
 
+        /**
+         * Adds the {@code Interaction} to the {@code Set<Interaction>} of the {@code Person} that we are building.
+         */
+        public PersonBuilder addInteraction(Interaction interaction) {
+            this.interactions.add(interaction);
+            return this;
+        }
+
```
Comment (dickongwd) [inline src/main/java/seedu/address/model/person/Person.java:269]: Should we keep it to `withInteractions` or do `addInteraction` instead?

Comment (zhyuhan) [reply src/main/java/seedu/address/model/person/Person.java:269]: I think it's good to have both for flexibility.

**Pull request merged**